src/handlers/issue-deduplication.ts

```diff
@@ -0,0 +1,114 @@
+import { IssueSimilaritySearchResult } from "../adapters/supabase/helpers/issues";
+import { Context } from "../types";
+import { IssuePayload } from "../types/payload";
+
+const MATCH_THRESHOLD = 0.95;
+const WARNING_THRESHOLD = 0.75;
```

Comment:
These should be configurable values. Can you see how configuration is implemented in the other plugins?

Reply:
Added, Now it fetches the values from the `context.config`